src/sec_certs/configuration.py

```diff
@@ -54,6 +54,10 @@ class Configuration(BaseSettings):
         "https://sec-certs.org/cc/cc.tar.gz",
         description="URL from where to fetch the latest full archive of fully processed CC dataset.",
     )
+    pp_latest_full_archive: AnyHttpUrl = Field(
+        "https://sec-certs.org/cc/pp.tar.gz",
+        description="URL from where to fetch the latest full archive of fully processed PP dataset.",
+    )
```

Comment:
The `pp_latest_snapshot` config also needs to change. It will no longer live on the `/static/` subdir. But have the same layout as the CC and FIPS datasets. Could you make the change pls?

Reply:
Sure, I wanted to discuss this first before changing this.